BridgeAppSDK/SBABridgeInfo.swift

```diff
@@ -91,6 +91,32 @@ public protocol SBABridgeInfo: SBASharedAppInfo, SBBBridgeInfoProtocol {
      do not check for a text user.
     */
     var disableTestUserCheck: Bool { get }
+    
+    /**
+     Show a back button in the navigation bar of ORKTaskViewController.
+     */
+    var showBackButtonInNavBar: Bool { get }
```

Comment:
I thought it could be that both are shown. I've seen that in other RK projects we've worked on and I can see how a designer might think that's a good idea. 

I thought it would be handy to put these options in a .plist, and I thought (mistakenly) BridgeInfo.plist was the right place.